src/coreclr/tools/aot/ILCompiler.Reflection.ReadyToRun/GCInfoTypes.cs

```diff
@@ -149,6 +149,11 @@ internal GcInfoTypes(Machine machine)
                     NUM_UNTRACKED_SLOTS_ENCBASE = 5;
                     REGISTER_DELTA_ENCBASE = 3;
                     break;
+                case (Machine) 0x6264: /* LoongArch64 */
```

Comment:
System.Reflection.Metadata version was bumped in https://github.com/dotnet/runtime/pull/85738. Could you please clean this up to use the named enum values?

Reply:
OK, thanks.
I will do it now.